kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/NetConfigManager.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
```

Comment:
I believe we should define a provisional (`x-internal`) API [1] in the bundle`org.eclipse.kura.linux.net`.

As this service interface serializes and deserializes a NetInterfaceConfig to the OS network configuration files (Debian `ifupdown`, Fedora `network-scripts` etc), a better name could be NetInterfaceConfigSerializationService. Add Javadoc explaining the purpose of this service.

[1] http://www.vogella.com/tutorials/OSGi/article.html